packages/altive_lints/lint_test/lints/avoid_shrink_wrap_in_list_view.dart

```diff
@@ -1,3 +1,4 @@
+// ignore_for_file: avoid_single_child
```

Comment:
I have added ignore comments to suppress warnings from tests that are not related to the rules added this time.